source/manual/get-started.html.md

```diff
@@ -17,24 +17,278 @@ If you're having trouble with this guide, you can ask your colleagues or the #go
+perform operations with AWS on the command-line.
+
+1. [Sign in to the gds-users AWS Console](https://gds-users.signin.aws.amazon.com/console).
+1. Click on your email address in the top right.
+1. Click 'My Security Credentials'.
+1. Click 'Create access key'.
+1. Copy/paste them into the inputs that the gds-cli provides for you,
+   if you're following [the setup instructions](#first-run).
+
+#### Changing your MFA device
+
+1. Follow steps 1 to 7 in [set up your MFA][#set-up-your-mfa]
+1. Choose one of the two options (Remove or Resync)
+
+### Get the appropriate access
+
+An account in AWS doesn't give you access to anything, you'll need to be given rights.
+
+Add yourself to a lists of users found in [the data for the infra-security project][infra-terra]. There are 5 groups:
+
+- `govuk-administrators`: people in Reliability Engineering who are working on GOV.UK infrastructure
+- `govuk-internal-administrators`: people in GOV.UK who are working on GOV.UK infrastructure including Architects, Lead Developers and anyone else working on the AWS migration
+- `govuk-powerusers`: anyone else who can have production access on GOV.UK
+- `govuk-platformhealth-powerusers`: as above but for members of the GOV.UK Platform Health team
+- `govuk-users`: anyone else who needs integration access on GOV.UK
+
+> Note: There is a limit on the number of people that can be in each group. If you find that the limit has been hit, try and identify any users who no longer need access and can be removed. Otherwise, a new group will need to be created.
+
+The identifier you need to add is called the "User ARN". You can find this by going
+to the [users page in AWS IAM][iam] and selecting your profile.
+
+```
+arn:aws:iam::<account-id>:user/<firstname.lastname>@digital.cabinet-office.gov.uk
+```
+
+After your PR has been merged, someone from the `govuk-administrators`
+or `govuk-internal-administrators` group needs to deploy the
+`infra-security` project.
+
+## 6. Use your AWS access
+
+### First run
+
+Here, you will use the GDS CLI you installed and set up earlier. You'll be asked for AWS credentials on the first run:
+
+```shell
+$ gds aws govuk-integration-poweruser -l
+Welcome to the GDS CLI! We will now store your AWS credentials in the keychain using aws-vault.
+Enter Access Key ID: AK-YOUR-ACCESS-KEY-ID
+Enter Secret Access Key: blah blah
+Added credentials to profile "gds-users" in vault
+Successfully initialised gds-cli
+Enter token for arn:aws:iam::123456789012:mfa/firstname.lastname@digital.cabinet-office.gov.uk: 123456
+```
+
+Your (Secret) Access Key is from the AWS console. Follow [the instructions to generate one](#generate-a-pair-of-access-keys). The token requested at the end is the [MFA token](#set-up-your-mfa). If you have a GDS-issued Yubikey (you probably don't at this stage), set `gds config yubikey true` and the GDS CLI will automatically pull the MFA code from your Yubikey.
+
+![AWS Access Key](images/aws/access-key.png)
```

Comment:
This image doesn't exist

Reply:
Ah! It's in `source/images/aws/access-key.png` rather than `source/manual/images...`.

Reply:
Hmm, let's just delete this.